Ohana/Classes/Common/DataProviders/OHABAddressBookContactsDataProvider.h

```diff
@@ -49,6 +49,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (instancetype)init NS_UNAVAILABLE;
 
++ (OHABAddressBookContactsDataProvider *)new NS_UNAVAILABLE;
```

Comment:
Sure @NickEntin, I'll create new PR for adding `+ (instancetype)new NS_UNAVAILABLE;`